src/calibration.cpp

```diff
@@ -135,6 +135,16 @@ void Calibration::cancelCalibration()
 	m_cancel = true;
 }
 
+const bool Calibration::hasContextCalibration()
```

Comment:
I was actually referring to the method signature here `bool Calibration::hasContextCalibration() const`